packages/beacon-node/src/chain/opPools/aggregatedAttestationPool.ts

```diff
@@ -387,18 +372,16 @@ export class AggregatedAttestationPool {
             sameAttDataCons[i].byCommittee.set(committeeIndex, attestationNonParticipation);
             sameAttDataCons[i].totalNotSeenCount += attestationNonParticipation.notSeenAttesterCount;
           }
-          for (const consolidation of sameAttDataCons) {
-            const score = consolidation.totalNotSeenCount / slotDelta;
-            if (score < minScore) {
-              minScore = score;
-            }
+        } // all committees are processed
 
-            consolidations.set(consolidation, score);
+        // after all committees are processed, we have a list of sameAttDataCons
```

Comment:
this for loop is the same except for removing "minScore" logic, it was misplaced at line 390 above inside this loop `for (const [committeeIndex, attestationGroup] of attestationGroupByIndex.entries()) {`